src/controller/CHIPDeviceController.cpp

```diff
@@ -227,47 +231,90 @@ CHIP_ERROR DeviceController::Init(NodeId localDeviceId, ControllerInitParams par
     return err;
 }
 
-CHIP_ERROR DeviceController::LoadLocalCredentials(Transport::AdminPairingInfo * admin)
+CHIP_ERROR DeviceController::GenerateOperationalCertificates(const ByteSpan & CSR, NodeId deviceId, MutableByteSpan & cert)
 {
-    ChipLogProgress(Controller, "Getting operational keys");
-    Crypto::P256Keypair * keypair = admin->GetOperationalKey();
+    // This code requires about 2K RAM to generate the certificates.
+    // The code would run as part of commissioner applications, so RAM requirements should be fine.
+    // Need to analyze if this requirement could be better managed by using static memory pools.
```

Comment:
A take slight issue with this assertion.  This is also called by autonomous controllers to generate certificates for themselves.  So that's just any controller that can bootstrap itself onto the fabric using ecosystem-specific mechanisms, as opposed to nodes that are commissioned by other Matter nodes.

I agree that such nodes are probably not constrained.  But they don't necessarily need to be commissioners.